pydm/widgets/timeplot_curve_editor.py

```diff
@@ -0,0 +1,71 @@
+from ..PyQt.QtGui import QDialog, QVBoxLayout, QHBoxLayout, QTableView, QAbstractItemView, QSpacerItem, QSizePolicy, QDialogButtonBox, QPushButton, QItemSelection
+from ..PyQt.QtCore import Qt, pyqtSlot
```

Comment:
@hhslepicka I am currently using PyQt 5.7 (with Python 3.5). You are right, importing QItemSelection differs in PyQt4 and PyQt5. I agree that a version checking would be helpful here. :)